nexus/src/authn/silos.rs

```diff
@@ -403,7 +403,16 @@ impl SamlIdentityProvider {
                                     {
                                         // Read comma separated group names
                                         for group in value.split(',') {
-                                            groups.push(group.to_string());
+                                            // Trim whitespace
```

Comment:
I imagine we want to trim the whitespace after splitting in case there's whitespace after commas.

Reply:
I think that's what's happening in that code?

Reply:
You're right.  I totally misread what was happening there.  Sorry!